ql-mbim-core.c

```diff
@@ -614,10 +614,10 @@ int mbim_reboot_modem(void)
          * 2. call quectel-modemfwd-helper --reboot
          *
          * In fact, when step1 finished, the module reboot itself automatically.
-         * Return 1 to the caller pretending the call is successful.
+         * Return success value to the caller pretending the call is successful.
          */
         info_printf("Could not find a Quectel modem available for commands!\n");
-        return 1;
+        return 0;
```

Comment:
success and non-success values can use macros to maintain consistency across the code. This needs to be modified by defining below two macros in some common header files. 
#define _EXIT_SUCCESS 0
#define _EXIT_FAILURE -1